mvnw.cmd

```diff
@@ -0,0 +1,187 @@
+@REM ----------------------------------------------------------------------------

```

Comment:
Have you tested the build on Windows?

Reply:
We don't currently support building on windows, so I'd be surprised if this works at all.